### PR TITLE
#120: API-core-initialize_finalize-InitializationSettings from md to rst

### PR DESCRIPTION
**Pull request by pierrepebay**
### THIS PR:
* API/core/initialize_finalize/InitializationSettings transition from .md to .rst

before (.md) | after (.rst)
:----------------: | :-------------: 
image | image

### CODE REVIEW

**Comment by pierrepebay** [comment]
I moved the class function explanations that used to be plain text inside the class interface section with sphinx syntax.

**Review by fnrizzi — APPROVED** [review]

File: docs/source/API/core/initialize_finalize/InitializationSettings.rst [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+    auto settings = Kokkos::InitializationSettings()
+                    .set_num_threads(8)
+                    .set_device_id(0)
+                    .set_disable_warnings(false);
+
+``InitializationSettings`` is a class that can be used to define the settings for
+initializating Kokkos programmatically without having to call the two parameter
+form (``argc`` and ``argv``) of `Kokkos::initialize() <initialize.html#kokkosinitialize>`_.
+It was introduced in version 3.7 as a replacement for the
```
Comment (nmm0) [inline docs/source/API/core/initialize_finalize/InitializationSettings.rst:25]: Can you use `versionadded`? https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded

**Review by nmm0 — CHANGES_REQUESTED** [review]
Should be good besides one minor change!

**Review by fnrizzi — CHANGES_REQUESTED** [review]
as per our 1:1 discussion

File: docs/source/API/core/initialize_finalize/InitializationSettings.rst (hunk 1 above)
Comment (fnrizzi) [reply docs/source/API/core/initialize_finalize/InitializationSettings.rst]: fixed ,can you please check? @nmm0

**Review by fnrizzi — APPROVED** [review]

**Review by nmm0 — APPROVED** [review]
Looks great!

**Pull request merged**